src/features/api-keys/lib.ts

```diff
@@ -12,6 +12,7 @@ export interface ApiKeyOptions {
 	name?: string;
 	description?: string;
 	tx?: Tx;
+	expiryDate?: string;
```

Comment:
NIT: Why don't we move this below `description` property so that it's closer to the props that are part of the api key model?